bin/needs-a-genny

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env ruby
 
+EXERCISES = File.expand_path('../../exercises', __FILE__)
```

Comment:
a reminder that this is equivalent to the following:

```ruby
EXERCISES = File.expand_path('../exercises', __dir__)
```

a suggestion that it is better to use this because the extra set of `..` can otherwise be confusing. If you will allow the use of the logical fallacy called "appeal to authority", note that rails did this in https://github.com/rails/rails/commit/5b8738c2df003a96f0e490c43559747618d10f5f, citing that using `__FILE__` hurts the person's soul and `__dir__` was done to "restore order"

Reply:
```suggestion
EXERCISES = File.expand_path('../exercises', __dir__)
```

Reply:
Thank you. Not a reminder, but rather the first time I've seen this. I sure wish the docs for File highlighted this feature...

Reply:
I believe it is 'appeal to irrelevant authority'.  In this case your authority is not irrelevant to code, but they are probably not a relevant authority on the soul (but who knows... porbably depends on how much coffee they drink) :smiling_imp:

Reply:
PS. I was very confused by the double `..`, so I much prefer this option.

Reply:
Implemented.